app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleRouter.kt

```diff
@@ -126,30 +170,83 @@ fun SearchPeopleContent(
-                        SearchAllPeopleScreen(
+                    var focusedTabIndex: Int by remember { mutableStateOf(initialPageIndex) }
+                    val keyboardController = LocalSoftwareKeyboardController.current
+                    val focusManager = LocalFocusManager.current
+
+                    if (screenType == SearchPeopleScreenType.CONVERSATION_DETAILS) {
+                        CompositionLocalProvider(LocalOverscrollConfiguration provides null) {
+                            HorizontalPager(
+                                state = pagerState,
+                                count = SearchPeopleTabItem.values().size,
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                            ) { pageIndex ->
+                                when (SearchPeopleTabItem.values()[pageIndex]) {
+                                    SearchPeopleTabItem.PEOPLE -> {
+                                        GetPeopleScreen(
+                                            isSearchActive = isSearchActive,
+                                            searchQuery = searchQuery.text,
+                                            noneSearchSucceed = noneSearchSucceed,
+                                            searchResult = searchResult,
+                                            contactsAddedToGroup = contactsAddedToGroup,
+                                            onAddContactToGroup = onAddContactToGroup,
+                                            onRemoveContactFromGroup = onRemoveContactFromGroup,
+                                            onOpenUserProfile = onOpenUserProfile,
+                                            onAddContact = onAddContact,
+                                            lazyListState = lazyListStates[pageIndex],
+                                            initialContacts = initialContacts
+                                        )
+                                    }
+
+                                    SearchPeopleTabItem.SERVICES -> SearchAllServicesScreen(
+                                        searchQuery = searchQuery.text,
+                                        searchResult = servicesSearchResult,
+                                        initialServices = servicesInitialContacts,
+                                        onServiceClicked = onServiceClicked,
+                                        lazyListState = lazyListStates[pageIndex]
+                                    )
+                                }
+                            }
+
+                            LaunchedEffect(pagerState.isScrollInProgress, focusedTabIndex, pagerState.currentPage) {
+                                if (!pagerState.isScrollInProgress && focusedTabIndex != pagerState.currentPage) {
+                                    keyboardController?.hide()
+                                    focusManager.clearFocus()
```

Comment:
Do we want to clear the focus when switching these tabs? The search input is above the pager so it shouldn't be a problem and wouldn't make the user to click again to regain the focus and continue writing after changing the tab 🤔

Reply:
Ok, it can be both pro and con depending on how the user uses this screen 😄 Changing it requires only removing these two lines of code so maybe you're right and we should just leave it as it is now and change it if it doesn't pass the design review or users start pointing it out 😉